integration-tests/test/nft_bridge.spec.ts

```diff
@@ -41,55 +41,6 @@ describe('NFT Bridge Test', async () => {
     )
   })
 
-  describe('Ownership tests', async () => {
```

Comment:
these were moved to unit tests of packages/boba/contracts, we don't need the whole infra up for these

Reply:
Sounds good!